src/PrestaShopBundle/Form/Admin/Sell/Product/Description/DescriptionType.php

```diff
@@ -77,6 +77,23 @@ public function __construct(
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $productId = (int) $options['product_id'];
+        $builder
+            ->add('packed_products', EntitySearchInputType::class, [
+                'label' => $this->trans('Pack of products', 'Admin.Catalog.Feature'),
+                'label_tag_name' => 'h2',
+                'entry_type' => PackedProductType::class,
+                'entry_options' => [
+                    'block_prefix' => 'packed',
+                ],
+                'remote_url' => $this->router->generate('admin_products_v2_search_combinations_for_packed', [
+                    'languageCode' => $this->employeeIsoCode,
+                    'query' => '__QUERY__',
+                ]),
+                'min_length' => 3,
+                'filtered_identities' => [],
```

Comment:
```suggestion
                'filtered_identities' => $productId > 0 ? [$productId] : [],
```

Reply:
This prevents the product from assigning itself

Reply:
we make the filter on the new field unique_identifier so I have to change this line but with an array of unique_identifier

Reply:
comment based on outdated code .... I not it as resolved